velox/connectors/hive/HiveConnectorSplit.h

```diff
@@ -22,14 +22,28 @@
 
 namespace facebook::velox::connector::hive {
 
+struct SortingColumn {
+  const std::string column;
+  const bool ascending{true};
+
+  SortingColumn(const std::string& _column, bool _ascending)
+      : column(_column), ascending(_ascending) {}
+};
+
+enum class BucketFunction { kNative, kHive };
+
 struct HiveConnectorSplit : public connector::ConnectorSplit {
   const std::string filePath;
   dwio::common::FileFormat fileFormat;
   const uint64_t start;
   const uint64_t length;
   const std::unordered_map<std::string, std::optional<std::string>>
       partitionKeys;
-  std::optional<int32_t> tableBucketNumber;
+  const std::optional<int32_t> tableBucketNumber;
```

Comment:
These are table-level properties, hence, they belong to HiveTableHandle, not HiveConnectorSplit.